AsyncDisplayKit/Layout/ASStackLayoutChild.h

```diff
@@ -0,0 +1,22 @@
+/*
+ *  Copyright (c) 2015-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+/**
+ Each child may override their parent stack's cross axis alignment.
+ @see ASStackLayoutAlignItems
+ */
+typedef NS_ENUM(NSUInteger, ASStackLayoutAlignSelf) {
```

Comment:
But `ASStackLayoutAlignSelf` is required in ASLayoutable, which is imported by layout specs and especially ASDisplayNode. So if we put this type in ASStackLayoutSpec.h, other stack-related types will be imported as well, and they are irrelevant unless stack layout is used.